src/app/public/modules/autocomplete/autocomplete.component.spec.ts

```diff
@@ -18,17 +18,19 @@ import { SkyAutocompleteComponent } from './autocomplete.component';
 import { SkyAutocompleteInputDirective } from './autocomplete-input.directive';
 
 import { SkyAutocompleteFixturesModule } from './fixtures/autocomplete-fixtures.module';
-import { SkyAutocompleteTestComponent } from './fixtures/autocomplete.component.fixture';
+import { SkyAutocompleteFixtureComponent } from './fixtures/autocomplete.component.fixture';
 
 import {
   SkyAutocompleteSearchFunction
 } from './types';
+import { SkyAutocompleteReactiveFixtureComponent } from './fixtures/autocomplete-reactive.component.fixture';
 
 describe('Autocomplete component', () => {
-  let fixture: ComponentFixture<SkyAutocompleteTestComponent>;
-  let component: SkyAutocompleteTestComponent;
+  let fixture: ComponentFixture<any>;
```

Comment:
Why are we switching these?

Reply:
Dang, I knew you'd point this out :)

Well, I made a different fixture to test reactive vs template-driven. I took a short cut here to avoid creating a whole new outer `describe` block with different setup. Its set to `any` so I can create a new testbed over it in line `171`.

Think its better to split them out?

Reply:
Ya. I now see that you are even destroying the created component from the outer describe and then remaking it in the reactive describe block. I'd rather us add a `beforeEach` to the template driven describe block and then just redeclare these two variables in both blocks.

Reply:
done